docs/blog/seeed/v0.2.10.md

```diff
@@ -0,0 +1,38 @@
+![Seeed Grove Zero](/static/blog/seeed/SeeedTarget.png)
+# Microsoft MakeCode for Seeed Grove Zero
+
+**Posted on November 14, 2017 by [riknoll](https://github.com/riknoll)**
+
+The Microsoft MakeCode Team is excited to announce the beta release of an editor for the [Seeed Grove Zero](https://www.seeedstudio.com/edu/grove-zero.html) system!
+
+Grove Zero is made up of magnetic modules that snap together for endless possibilities. Program
+the main board to control all of the various inputs and outputs out of the box. The magnetic connectors
```

Comment:
Are the new lines here intended? Would this break up the line in markdown?

Reply:
Yep, intended to make the .md more readable. Markdown requires a double new line for a new line to show up in the rendered page.

Reply:
Ah makes sense :)